fate/src/main/java/org/apache/accumulo/fate/util/UtilWaitThread.java

```diff
@@ -29,4 +33,36 @@ public static void sleep(long millis) {
       log.error("{}", e.getMessage(), e);
     }
   }
+
+  /**
+   * Copied from Guava release 23. The Uniterruptibles class was annotated as Beta by Google, therefore unstable to use. The following javadoc was copied from
+   * com.google.common.util.concurrent.Uninterruptibles:
+   *
+   * Utilities for treating interruptible operations as uninterruptible. In all cases, if a thread is interrupted during such a call, the call continues to
+   * block until the result is available or the timeout elapses, and only then re-interrupts the thread.
+   *
+   * @author Anthony Zana
+   * @since 10.0
+   */
+  public static void sleepUninterruptibly(long sleepFor, TimeUnit unit) {
```

Comment:
It may be nice to put all copied guava code into one package.  Not sure if this is possible.